src/app/views/app-sections/AppTitle.tsx

```diff
@@ -53,8 +55,9 @@ export function appTitleDisplayOnMobileScreen(
       <div style={{ padding: 10 }} role={'heading'} aria-level={1}>
         <Label className={classes.graphExplorerLabel}>
           Graph Explorer
-          </Label>
+          <Label className={classes.versionLabel}>{`V${getVersion()}`}</Label>
```

Comment:
This can be written once at the top and called in each function. It will make it easier in case there will be a need for a change in the future